src/coreclr/tools/SuperFileCheck/SuperFileCheck.csproj

```diff
@@ -44,7 +44,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="$(MicrosoftCodeAnalysisCSharpVersion)" />
-    <PackageReference Include="runtime.$(JITToolsRid).Microsoft.NETCore.Runtime.JIT.Tools">
+    <PackageReference Condition="'$(DotNetBuildFromSource)' != 'true'" Include="runtime.$(JITToolsRid).Microsoft.NETCore.Runtime.JIT.Tools">
```

Comment:
At first I was thinking this project should be excluded from source-build but I guess someone could retrieve LLVM's FileCheck and still use the tool.